### DISCUSSION TITLE
UI/UX 設計超人氣軟體 Figma，打開了中美科技戰 2.0 的新戰場

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#625] [General]
https://buzzorange.com/techorange/2022/05/03/dji-figma-russia-ukraine/

在烏俄戰爭之中，我們可以看見新的科技力量，例如無人機所發揮的戰爭價值。然而，現在這場烏俄戰爭，其實也能看出中美之間的角力戰，現在進入了「科技戰 2.0」階段，而打開這場戰役的，竟是 UI/UX 設計界的超人氣軟體——Figma。

原因是美軟體霸主 Figma 對中國斷供！中國 UI 圈炸鍋

**Comment by HsuChen88** [comment]
題外話 所以大疆在這場戰爭中賺了兩邊的錢
